src/pkg/scan/sbom/sbom.go

```diff
@@ -164,3 +196,146 @@ func retrieveSBOMContent(rawReport string) ([]byte, *v1.Scanner, error) {
 	}
```

Comment:
line156 on the left and line 188 on the right, should the `rpt := vuln.Report{}` reference a struct within `pkg/scan/sbom` folder?